image/src/utils/ipfs.ts

```diff
@@ -1,9 +1,28 @@
+import { $purify, getProviderList } from '@kodadot1/minipfs'
+
 type FetchIPFS = {
   path: string
   gateway1: string
   gateway2: string
 }
 
+export function toNftstorage(path: string) {
+  const infura = new URL(getProviderList(['nftstorage'])[0])
+  const url = new URL(path)
+  url.hostname = infura.hostname
+
+  return url.toString()
+}
+
+export function ipfsUrl(ipfs?: string) {
+  if (!ipfs) {
+    return ''
+  }
+
+  // TODO: 'kodadot_beta' for beta, 'kodadot' for prod
+  return $purify(ipfs, ['kodadot_beta'])[0]
```

Comment:
```suggestion
  return purifySingle(ipfs, 'kodadot_beta')
```

Reply:
Needs to be implemented on the side of `minipfs`

Reply:
for `minipfs` I created this issue for followup PRs https://github.com/kodadot/packages/issues/182